hack/lib/util.sh

```diff
@@ -222,18 +239,18 @@ kube::util::gen-docs() {
   "${genyaml}" "${dest}/docs/yaml/kubectl/"
 
   # create the list of generated files
-  pushd "${dest}" > /dev/null
+  pushd "${dest}" > /dev/null || exit 1
```

Comment:
`return 1` and a bunch of others still have `exit 1`

Reply:
Ok, I'll change all instances of `exit 1` in this context to `return 1`. We can still think of the alternatives I mentioned in my other comment.